mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
+
+msgid "menu_secondary_select_key"
+msgstr "Tecla Cancelar"
+
+msgid "spyder_route4_wulf2"
+msgstr "Ooh, ¡me siento tan castrado!"
+
+msgid "menu_left_key"
+msgstr "Tecla Izquierda"
+
+msgid "dark_robo"
+msgstr "Robo Oscuro"
+
+msgid "spyder_citypark_maniac"
+msgstr "Phew, ¡lo hemos cagado!"
+
+msgid "dodge_increased"
+msgstr "el {stat} de {user} ha incrementado."
+
+msgid "menu_up_key"
+msgstr "Tecla Subir"
+
+msgid "menu_right_key"
+msgstr "Tecla Derecha"
+
+msgid "menu_primary_select_key"
+msgstr "Tecla Acción"
+
+msgid "menu_back_key"
+msgstr "Tecla Volver"
+
+msgid "options_new_input_key0"
+msgstr "Pulse una tecla para fijar su tecla de entrada nueva"
```

Comment:
Too literal. I don't like the keyboard assumption here, but it is also in the original sentence:

```suggestion
msgstr "Pulsa una tecla para establecerla como nuevo valor de la entrada"
```